xray/core/common.py

```diff
@@ -272,6 +272,31 @@ def groupby(self, group, squeeze=True):
             group = self[group]
         return self.groupby_cls(self, group, squeeze=squeeze)
 
+    def rolling(self, min_periods=None, center=False, **kwarg):
+        """Returns a Rolling object for performing moving window operations.
+
+        Parameters
+        ----------
+        min_periods : int, default None
+            Minimum number of observations in window required to have a value
+            (otherwise result is NA).
+        center : boolean, default False
+            Set the labels at the center of the window.
+        kwarg : dim=window
+            dim : str
```

Comment:
just checked, not perfect.


![temp](https://cloud.githubusercontent.com/assets/2443309/11547094/3e4e9a8e-9906-11e5-8d1b-f956fecce81b.png)